core/utils/arp.h

```diff
@@ -0,0 +1,42 @@
+#ifndef BESS_UTILS_ARP_H_
+#define BESS_UTILS_ARP_H_
+
+#include "endian.h"
+#include "ether.h"
+#include "ip.h"
+
+namespace bess {
+namespace utils {
+
+// A basic ARP header definition.
+struct[[gnu::packed]] ARP {
+  // Ethernet hardware format for hrd
+  static const be16_t kEtherHardwareFormat = 1;
+
+  enum Opcode : uint16_t {
+    REQUEST = 1,
```

Comment:
Let's use the `kRequest` style, for consistency with other protocol header files in `utils/`